packages/create-web-scripts-library/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "@spotify/web-scripts/config/tsconfig.json",
-  "include": ["src"]
+  "include": ["src"],
+  "compilerOptions": {
+    "skipLibCheck": true
```

Comment:
Commander has a broken type in latest 2.x. We should upgrade to 4.x, which has working types, but breaking changes. https://github.com/spotify/web-scripts/issues/95